gym/envs/__init__.py

```diff
@@ -78,6 +78,13 @@
     reward_threshold=90.0,
 )
 
+register(
```

Comment:
when registered this way, `MountainCarContinuous-v1` and `MountainCarContinuous-v0` are the exact same environment